go-controller/pkg/ovn/policy.go

```diff
@@ -43,6 +46,64 @@ func (oc *Controller) syncNetworkPoliciesPortGroup(
 	}
 }
 
+func (oc *Controller) getACLUUIDFromMap(acls map[string]interface{}, filters map[string]interface{}) string {
+	for aclUUID, acl := range acls {
+		found := true
+		for filterID, filter := range filters {
+			if filterID == "external_ids" {
+				if externalIds, ok := acl.(map[string]interface{})["external_ids"]; ok {
+					for key, val := range filter.(map[string]string) {
+						externalID := externalIds.(map[string]interface{})[key]
+						if externalID == nil || externalID.(string) != val {
+							found = false
+							break
+						}
+					}
+					if !found {
+						break
+					}
+				} else {
+					found = false
+					break
+				}
+			} else if acl.(map[string]interface{})[filterID] != filter {
+				found = false
+				break
+			}
+		}
+
+		if found {
+			return aclUUID
+		}
+	}
+
+	return ""
+}
+
+func (oc *Controller) insertACL(row map[string]interface{}, portGroupUUID string) error {
+	var err error
+
+	retry := true
+	for retry {
+		txn := oc.ovnNBDB.Transaction("OVN_Northbound")
+		newACLID := txn.Insert(dbtransaction.Insert{
+			Table: "ACL",
+			Row:   row,
+		})
+		txn.InsertReferences(dbtransaction.InsertReferences{
+			Table:           "Port_Group",
+			WhereId:         portGroupUUID,
+			ReferenceColumn: "acls",
+			InsertIdsList:   []string{newACLID},
```

Comment:
Should not InsertIdsList include all the previously inserted ACLs here?  newACLID is only one string right? Looks like I have forgotten how OVSDB insert and wait is supposed to work.